libs/features/mas/web-components/src/merch-datasource.js

```diff
@@ -52,23 +53,25 @@ async function parseMerchCard(fragmentData, appendFn, merchCard, consonant) {
     const { variant = 'catalog' } = item;
     merchCard.setAttribute('variant', variant);
     const cardMapping = cardContent[variant] ?? 'catalog';
-    item.icon?.forEach((icon) => {
+    item.mnemonicIcon?.forEach((icon, idx) => {
+        const href = item.mnemonicLink?.length > idx ? item.mnemonicLink[idx] : '';
+        const alt = item.mnemonicAlt?.length > idx ? item.mnemonicAlt[idx] : '';
         const merchIcon = createTag('merch-icon', {
             slot: 'icons',
             src: icon,
-            alt: '',
-            href: '',
+            alt,
```

Comment:
if alt and href are undefined, are they properly omitted or rendered as `alt="undefined"` ? I couldn't verify it.

Reply:
good point, will test it today